### PR TITLE
Preview - Sveltekit/Svelte 5

### PR DESCRIPTION
**Pull request by birkskyum**
**SvelteKit /w Svelte 4** ("sveltekit-4")
I took the existing Svelte 4 , and ported it to SvelteKit SSG, still with Svelte 4 - because it's the recommended way to build with Svelte (and easier to upgrade/configure).

**SvelteKit /w Svelte 5 (5.0.0-next.1)** ("sveltekit-5")
I then upgraded to Svelte 5 (still without runes), and **partialUpdate** broke, so I had to change the following to again trigger change detection, and I believe that's why this performs worse in partialUpdate test. ( fyi @trueadm - don't know if this is expected ).

```ts
clone[i].label += " !!!";
```

changed to

```ts
clone[i] = { ...clone[i], label: clone[i].label + " !!!" };
````

**Runes** ("sveltekit-5-runes")
I then enabled runes mode, and updated to that syntax (and reverted the prev. change to partialUpdates):
```ts
  let rowId = $state(1);
  let data = $state([]);
  let selected = $state(undefined);
```

**Runes Fine Gained** ("sveltekit-5-runes-fine-grained")
Finally, I also made each label a rune:

```ts
let label = $state(`${adjectives[_random(adjectives.length)]} ${
      colours[_random(colours.length)]
    } ${nouns[_random(nouns.length)]}`);
```

Screenshot 2023-11-11 at 23 18 34

### CODE REVIEW

**Comment by trueadm** [comment]
I've already put up a Svelte 5 upgrade PR :) https://github.com/krausest/js-framework-benchmark/pull/1494

File: frameworks/keyed/sveltekit-5-runes-fine-grained/src/routes/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  function createItem() {
+    let label = $state(`${adjectives[_random(adjectives.length)]} ${
+      colours[_random(colours.length)]
+    } ${nouns[_random(nouns.length)]}`);
+
+    return {
+      id: rowId++,
+      label,
```
Comment (trueadm) [inline frameworks/keyed/sveltekit-5-runes-fine-grained/src/routes/+page.svelte:109]: This isn't fine-grain. It needs to be inside a getter/setter. See https://github.com/krausest/js-framework-benchmark/pull/1494 for how you can use `$state` on classes instead.

**Comment by trueadm** [comment]
I'm also not sure we need to bring along SvelteKit here too.

**Comment by birkskyum** [comment]
Excellent, thanks! :)

**Pull request closed without merging**